src/main/components/form/claimDetailsValidator.ts

```diff
@@ -0,0 +1,70 @@
+export const validatePersonalDetails = (userCase: Record<string, any>): boolean => {
+  if (!userCase) {
+    return false;
+  }
+
+  const {
+    typeOfClaim,
+    reasonableAdjustments,
+    personalDetailsCheck,
+    hearingPreferences,
+    claimantContactPreference,
+    claimantContactLanguagePreference,
+    claimantHearingLanguagePreference,
+    address1,
+    addressTown,
+    addressPostcode,
+  } = userCase;
+
+  if (
+    !typeOfClaim ||
+    !reasonableAdjustments ||
+    !personalDetailsCheck ||
+    !hearingPreferences ||
+    !claimantContactPreference ||
+    !claimantContactLanguagePreference ||
+    !claimantHearingLanguagePreference ||
+    !address1 ||
+    !addressTown ||
+    !addressPostcode
+  ) {
+    return false;
+  }
+
+  return true;
+};
+
+export const validateEmploymentAndRespondentDetails = (userCase: Record<string, any>): boolean => {
+  if (!userCase) {
+    return false;
+  }
+
+  const { startDate, pastEmployer, isStillWorking, claimantWorkAddressQuestion, respondentEnterPostcode, noticeEnds } =
+    userCase;
+
+  if (pastEmployer === 'Yes') {
+    if (!startDate) {
+      return false;
+    }
+  }
+  if (isStillWorking === 'Notice') {
+    if (!startDate || !noticeEnds) {
+      return false;
+    }
+  }
+  return !(!pastEmployer || isStillWorking === undefined || !claimantWorkAddressQuestion || !respondentEnterPostcode);
+};
+
+export const validateClaimCheckDetails = (userCase: Record<string, any>): boolean => {
+  if (!userCase) {
+    return false;
+  }
+
+  const { claimTypePay, claimSummaryText } = userCase;
+
+  if (!Array.isArray(claimTypePay) || claimTypePay.length === 0) {
+    return false;
+  }
+
+  return !(!claimSummaryText || claimSummaryText.trim() === '');
```

Comment:
No need to check pay. Should just be typeOfClaim, the describe what happened text box which is possibly claimSummaryText but also the document upload as C's might upload a document rather than type out their claim

<img width="678" alt="Screenshot 2025-04-10 at 09 15 24" src="https://github.com/user-attachments/assets/a168e57b-4989-42a1-9823-8fd8936c0e18" />